### PR TITLE
chore: fix DE locale typo for validation messages

### PR DESCRIPTION
**Pull request by Knorgias**
## What I did

1. Fixed a typo for DE locale on validation messages

### CODE REVIEW

File: packages/ui/components/validate-messages/translations/de.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ export default {
     MinLength: 'Du musst mindestens {params} Zeichen eingeben.',
     MaxLength: 'Du kannst maximal {params} Zeichen eingeben.',
     MinMaxLength: 'Du musst zwischen {params.min} und {params.max} Zeichen eingeben.',
-    Pattern: 'Geben Sie ein gültiges {fieldName} ein.',
-    IsNumber: 'Geben Sie ein gültiges {fieldName} ein.',
+    Pattern: 'Geben Sie ein gültige {fieldName} ein.',
```
Comment (gerjanvangeest) [inline packages/ui/components/validate-messages/translations/de.js:8]:
```suggestion
    Pattern: 'Geben Sie eine gültige {fieldName} ein.',
```

Also for all other translations.
But maybe good to let the requester of this change also review this.

**Comment by gerjanvangeest** [comment]
@Knorgias can you add a changeset?

File: packages/ui/components/validate-messages/translations/de.js [hunk 2]
```diff
@@ -5,13 +5,13 @@ export default {
...
+    IsNumber: 'Geben Sie ein gültige {fieldName} ein.',
     MinNumber: 'Geben Sie für {fieldName} einen Wert über {params} ein.',
     MaxNumber: 'Geben Sie für {fieldName} einen Wert unter {params} ein.',
     MinMaxNumber:
       'Geben Sie für {fieldName} einen Wert zwischen {params.min} und {params.max} ein.',
-    IsDate: 'Bitte geben Sie ein gültiges Datum ein (TT.MM.JJJJ).',
+    IsDate: 'Bitte geben Sie ein gültige Datum ein (TT.MM.JJJJ).',
```
Comment (tkgroot) [inline packages/ui/components/validate-messages/translations/de.js:14]: here 'gültiges' is correct. 
```suggestion
    IsDate: 'Bitte geben Sie ein gültiges Datum ein (TT.MM.JJJJ).',
```

File: packages/ui/components/validate-messages/translations/de.js [hunk 3]
```diff
@@ -27,12 +27,12 @@ export default {
...
     MaxLength: 'Du kannst maximal {params} Zeichen eingeben.',
     MinMaxLength: 'Du solltest zwischen {params.min} und {params.max} Zeichen eingeben.',
-    IsNumber: 'Geben Sie ein gültiges {fieldName} ein.',
+    IsNumber: 'Geben Sie ein gültige {fieldName} ein.',
     MinNumber: 'Geben Sie für {fieldName} einen Wert über {params} ein.',
     MaxNumber: 'Geben Sie für {fieldName} einen Wert unter {params} ein.',
     MinMaxNumber:
       'Geben Sie für {fieldName} einen Wert zwischen {params.min} und {params.max} ein.',
-    IsDate: 'Bitte geben Sie ein gültiges Datum ein (TT.MM.JJJJ).',
+    IsDate: 'Bitte geben Sie ein gültige Datum ein (TT.MM.JJJJ).',
```
Comment (tkgroot) [inline packages/ui/components/validate-messages/translations/de.js:35]:
```suggestion
    IsDate: 'Bitte geben Sie ein gültiges Datum ein (TT.MM.JJJJ).',
```

File: packages/ui/components/validate-messages/translations/de.js [hunk 4]
```diff
@@ -5,13 +5,13 @@ export default {
     MinLength: 'Du musst mindestens {params} Zeichen eingeben.',
     MaxLength: 'Du kannst maximal {params} Zeichen eingeben.',
     MinMaxLength: 'Du musst zwischen {params.min} und {params.max} Zeichen eingeben.',
-    Pattern: 'Geben Sie ein gültiges {fieldName} ein.',
-    IsNumber: 'Geben Sie ein gültiges {fieldName} ein.',
+    Pattern: 'Geben Sie ein gültige {fieldName} ein.',
+    IsNumber: 'Geben Sie ein gültige {fieldName} ein.',
```
Comment (tkgroot) [inline packages/ui/components/validate-messages/translations/de.js:9]: What kind of fieldName is expected here  for "IsNumber"?

**Review by tkgroot — CHANGES_REQUESTED** [review]
some corrections needed and unclarity on a translation

File: packages/ui/components/validate-messages/translations/de.js (hunk 4 above)
Comment (gerjanvangeest) [reply packages/ui/components/validate-messages/translations/de.js:9]: Fieldnames can be anything, e.g. amount, interest, number of guests.

Comment (tlouisse) [reply packages/ui/components/validate-messages/translations/de.js:9]: Coming up with generic German messages when we have a variable fieldName, seems to be impossible. Also see this discussion: https://github.com/ing-bank/lion/pull/1938#issuecomment-1468390611 
A good and easy way to mitigate this (for >99% of cases) would be https://github.com/ing-bank/lion/issues/1940

**Review by tkgroot — APPROVED** [review]

**Review by gerjanvangeest — APPROVED** [review]

**Pull request merged**